jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -2172,6 +2172,23 @@ properties:
             type: string
             fits_keyword: S_WHTLIT
             blend_table: True
+      resample:
+        title: Resample Parameters
+        type: object
+        properties:
+	  pointings:
```

Comment:
What's the big colored bar here? Does that indicate use of a TAB or something? Check the formatting.

Reply:
Oh, I know, the indenting is wrong. It needs to be indented another 4 spaces.